Bio/SeqAnalysis/SeqAnalyzer.py

```diff
@@ -0,0 +1,168 @@
+    'kot': 'WKQKKAQFDAQKEHQTSRNDIVTNSLEGKQTTEKFTERQECCCCCCCRVMGKKRAPQKGKTVTKPQEIIVDESKLNWKPVDIPDTLDDFGGFYGLEEIDGVDVKVVDGKVTFVTKKDSKVLKDSNKEKVGDDQESVENESGSDSESELLEFKNLDDIKEGELSAASYSSSDEDEQGNIESSKLTDPSEDVDEDVDEDIPIVEKLISNFSQK'
+}
+
+
+class Analyzer:
+    """
+    A class that takes in records from SeqAnalysis.database and performs
+    multiple operations on it and returns pd.DataFrame with values
+    """
+
+    def makedict(self, records) -> dict:
+        """
+        Makes a dictionary from records, where ids are keys and sequences are values
+
+        :return: returns a created dictionary
+        """
+        dict = {key: value.seq[0] for key, value in records}
+        return dict
+
+    def median_stdev_avg(self) -> tuple:
+        """
+        Calculates standard deviation of lenths of given sequences
+
+        :return: changes self.stdev values to standard value calculate from sequences
+        """
+        list_of_seq = self.dict.values()
+        list_of_lenths = [len(_) for _ in list_of_seq]
+        stdev = float(np.std(list_of_lenths))
+        avg = float(np.average(list_of_lenths))
+        median = float(np.median(list_of_lenths))
+        return stdev, avg, median
+
+    def seq_lenth(self) -> list:
+        """
+        Makes a list of sequences lenths
+
+        :return: returns a list of sequences lenths
+        """
+        return [len(_[1]) for _ in self.tuples]
+
+    def id(self) -> list:
+        """
+        Makes a list of protein IDs
+
+        :return: ruturns a list od protein ids
+        """
+        return [_[0] for _ in self.tuples]
+
+    def molecular_weight(self) -> tuple:
+        """
+        Main funcion calculating proteins values (10 values).
+
+        :return: Funcion returns a tuple containning:
+        1. molecular_weight - calculated malecular weight from protein sequences in a list
+        2. pi - calculated isoelectric point for protein sequences in a list
+        3. amino_acids - counted standard aa in protein sequences, in a list of dicts
+        4. aromacity - calculated aromacity for protein sequences according to Lobry 1994 method, in a list
+        5. amino_acids_percent - calculated percantage based aa content in protein sequences, in a list
+        6. instability - calculated instability indexes for protein sequences according to Guruprasad et al 1990, in a list
+        7. flexibility - calculated flexibility of protein sequences according to Vihinen, 1994, in a list
+        8. secondary_structure_fraction - calculated fraction of helix, turn and sheet for protein sequences, in a list
+        of tuples with 3 elements (helix, turn, sheet) fractions accordingly
+        9. seq_gravy - calculated gravy of protein sequences according to Kyte and Doolittle, in a list
+        10.mol_ext_coefficient - calculated molar extinction coefficients for protein sequences, in a list
+        """
+        l_objects = [ProteinAnalysis(_[1]) for _ in self.tuples]
```

Comment:
What does it mean `l_objects`?

Reply:
short for list of objects

Reply:
Please use self-explanatory and containing information names instead of generic types, so try to avoid words such as `tup`, `tuple`, `dict`, `obj`, `objects` etc